CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+## 🛑 Breaking changes 🛑
+
+- Deprecate `mem-ballast-size-mib` option in command line (#3626)
```

Comment:
Is not deprecated, is removed correct? Since the behavior is no longer there. Consider to explain what to do to fix this.

Reply:
Corrected the wordings. Thanks!